src/wordlift/external-plugin-hooks/avada-builder/class-avada-builder-support.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Wordlift\External_Plugin_Hooks\Avada_Builder;
+
+/**
+ * Avada Builder Support.
+ *
+ * @since 3.39.2
+ * @author Mahbub Hasan Imon <mahbub@wordlift.io>
+ */
+class Avada_Builder_Support {
+
+	public function __construct() {
+		add_filter( 'wl_post_excerpt_post_content', array( $this, 'post_excerpt_post_content' ), 10, 2 );
+	}
+
+	/**
+	 * Filter the post content before sending it to WordLift API.
+	 *
+	 * @param $post_body string The post content sent from WordPress editor.
+	 * @param $post_id int The post id.
+	 *
+	 * @return string The post content after removing Avada Builder's shortcodes.
+	 */
+	public function post_excerpt_post_content( $post_body, $post_id ) {
+
+		if ( $this->is_avada_builder_active( false, $post_id ) ) { // Check if its Avada Builder's Content.
+			return wp_strip_all_tags( do_shortcode( $post_body ), true );
+		}
+		return strip_shortcodes( $post_body );
```

Comment:
you are filtering the content, you need to return the original value if condition didnt match